rio_cogeo/scripts/cli.py

```diff
@@ -288,7 +301,13 @@ def info(input, to_json):
     {click.style("Alpha Band:", bold=True):<{sep}} {metadata['Profile']['Alpha Band']}
     {click.style("Internal Mask:", bold=True):<{sep}} {metadata['Profile']['Internal Mask']}
     {click.style("Interleave:", bold=True):<{sep}} {metadata['Profile']['Interleave']}
+    {click.style("ColorInterp:", bold=True):<{sep}} {metadata['Profile']['ColorInterp']}
     {click.style("ColorMap:", bold=True):<{sep}} {metadata['Profile']['ColorMap']}
```

Comment:
Maybe it would look better if ColorMap and ColorInterp are swapped

Reply:
Go for it